src/layer/ImageOverlay.js

```diff
@@ -165,6 +169,7 @@ export var ImageOverlay = Layer.extend({
 		img.onmousemove = Util.falseFn;
 
 		img.onload = Util.bind(this.fire, this, 'load');
+		img.onerror = Util.bind(this._overlayOnError, this, 'error');

```

Comment:
Yes, I basically mimicked the `'error'` from `TileLayer`.
I thought it was meant to be default error decription.